docs/android.md

```diff
@@ -0,0 +1,26 @@
+# Klaatu for Android
+
+The preferred method of running the android tests are using the provided docker image. The steps below outline how that can be done.
+
+## Docker
+
+Close your android studio instance if it is running. Navigate to your `platform-tools` folder where `adb` is located and run the following command:
+
+```sh
+./adb.exe -a nodaemon server start
```

Comment:
Does this work on Linux/macOS too? (re: that `.exe` extension).

Reply:
Good catch, lemme fix that